src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ScheduledActivity.java

```diff
@@ -0,0 +1,59 @@
+package ch.uzh.ifi.hase.soprafs21.entities;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+@Table(name = "ScheduledActivity")
+public class ScheduledActivity implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Column
+    private UserActivity chosenActivity;
+
+    @Column
+    private String chosenLocation;
+
+    @Column
+    private Date chosenDate;
```

Comment:
```suggestion
    private LocalDateTime chosenDate;
```
probably of type `LocalDateTime` containing date & time? If so, getter & setter need to be updated as well